app/models/statement.rb

```diff
@@ -53,6 +53,11 @@ def self.find_by_invoice_number(query)
     find_by(id: id, account_id: account_id)
   end
 
+  def self.where_invoice_number(query)
```

Comment:
Can we refactor `find_by_invoice_number` to use this method? I think something like this will do the trick?

```ruby
def self.find_by_invoice_number(query)
  where_invoice_number(query)&.first
end
```

Reply:
Sure I'll change it, and make sure that the test swipe runs